pkg/k8sutil/clientset.go

```diff
@@ -163,27 +163,7 @@ func GetKubeClient(ctx context.Context) (kbclient.Client, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to get cluster config")
 	}
-	kcli, err := kbclient.New(cfg, kbclient.Options{})
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to create kubebuilder client")
-	}
-	return kcli, nil
-}
-
-func GetVeleroKubeClient(ctx context.Context) (kbclient.Client, error) {
-	k8slogger := zap.New(func(o *zap.Options) {
-		o.DestWriter = io.Discard
-	})
-	log.SetLogger(k8slogger)
-	cfg, err := GetClusterConfig()
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to get cluster config")
-	}
-	scheme := runtime.NewScheme()
-	velerov1.AddToScheme(scheme)
-	kcli, err := kbclient.New(cfg, kbclient.Options{
-		Scheme: scheme,
-	})
+	kcli, err := kbclient.NewWithWatch(cfg, kbclient.Options{})
```

Comment:
We need the watch interface for our informer:
- https://github.com/replicatedhq/kots/blob/a422a554bd22d7b1e7e0feed4f76e33d77f0fc57/pkg/informers/informers.go#L48-L50